src/workflow/ArcanistDiffWorkflow.php

```diff
@@ -449,6 +450,9 @@ public function getArguments() {
 
   public function run() {
     $this->console = PhutilConsole::getConsole();
+    $this->refProvider = new UberRefProvider(
+      $this->configurationManager->getConfigFromAnySource('uber.arcanist.use_non_tag_refs', false)
```

Comment:
You will need to register this config value on `src/configuration/ArcanistSettings.php`.